model/src/main/resources/reference.conf

```diff
@@ -12,36 +12,54 @@ ort {
       }
     }
 
-    fileBasedStorage {
-      backend {
-        httpFileStorage {
-          url = "https://your-http-server"
-          headers {
-            key1 = "value1"
-            key2 = "value2"
+    options {
+      // A map of maps from scanner class names to scanner-specific key-value pairs.
+    }
+
+    storages {
+      local {
+        backend {
+          localFileStorage {
+            directory = ~/.ort/scanner/results
+            compression = false
           }
         }
+      }
 
-        localFileStorage {
-          directory = ~/.ort/scanner/results
-          compression = false
+      http {
+        backend {
+          httpFileStorage {
+            url = "https://your-http-server"
+            headers {
+              key1 = "value1"
+              key2 = "value2"
+            }
+          }
         }
       }
-    }
 
-    postgresStorage {
-      url = "jdbc:postgresql://your-postgresql-server:5444/your-database"
-      schema = schema
-      username = username
-      password = password
-      sslmode = "required"
-      sslcert = "/defaultdir/postgresql.crt"
-      sslkey = "/defaultdir/postgresql.pk8"
-      sslrootcert = "/defaultdir/root.crt"
-    }
+      clearlyDefined {
+        serverUrl = "http://clearly-defined.org"
```

Comment:
As we actually *can* specify something meaningful here, I believe we should, i.e. use "https://api.clearlydefined.io".